test/sharness/Makefile

```diff
@@ -14,6 +14,9 @@ IPFS_ROOT = ../..
 # User might want to override those on the command line
 GOFLAGS =
 
+# The targets in this makefile depend on being run sequentially
+.NOTPARALLEL:
```

Comment:
:+1: i would love to move towards being able to run them in parallel. wonder what git does + how it would manage the sharness output. cc @chriscool